src/style.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  background-color: gray;
+}
+
+.wrapper {
+  width: 50%;
+  background: white;
+}
+
+.line {
+  border: 0.2px solid #e8e8e9;
+}
+
+.wrap {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.9em 2em;
+  border-bottom: 0.2px solid #e8e8e9;
+}
+
+form {
+  display: flex;
+  justify-content: space-between;
+  padding: 24px;
+  border-bottom: 0.2px solid #e8e8e9;
+}
```

Comment:
Kindly use flexible measurements such as `rem`, `em`, and percentages to achieve responsiveness

Reply:
@amejid  Hi mejid, just saw the review, are you practicing or do you actually want me to make the change?